aiojenkins/jenkins.py

```diff
@@ -64,7 +65,7 @@ async def close(self) -> None:
         await self.session.close()
 
 
-class Jenkins:
+class Jenkins(AbstractAsyncContextManager):
```

Comment:
Subclassing this automatically provides the default `__aenter__` method (which returns `self`)